src/main/resources/application.yml

```diff
@@ -47,6 +48,7 @@ spring:
    rest:
       # this points at the local WireMock server in the test environment.
       iexBasePath: http://localhost:${wiremock.server.port}/
+      iexToken: pk_8bf2fa8153b64102b1d34199273d2063/
```

Comment:
Dont' commit secrets to code repos, keep them in local config files that do not get committed